server/index.js

```diff
@@ -56,6 +56,110 @@ app.get('/api/products/:productId', (req, res, next) => {
   }
 });
 
+app.get('/api/cart', (req, res, next) => {
+  const sqlEmpty = `
+  SELECT *
+  FROM "carts"
+  `;
```

Comment:
You don't need to query the database if there is no `req.session.cartId`. Just respond with an empty array. It doesn't make sense to give the client every user's cart items.